mgmtsystem_security_event/views/essential_assets.xml

```diff
@@ -0,0 +1,44 @@
+<openerp>
```

Comment:
Please use the full model name for this file `mgmtsystem_security_assets_essential.xml`